docs/topics/proc_debug-logging-nodejs.adoc

```diff
@@ -0,0 +1,162 @@
+
+. Test your application to invoke debug logging. 
++
+For example, if you had debug logging in the xref:mission-http-api-nodejs[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl http://localhost:8080/api/greeting?name=Sarah
+----
+
+. View your application logs to see your debug messages.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+mybooster name: Sarah +3m
+----
+
+
+== Show Debug Logging on OpenShift
+
+Use the the `DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+[discrete]
+=== Prerequisites 
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+[discrete]
+=== Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm run openshift
+----
++
+This runs the `openshift` `npm` script, which wraps direct calls to xref:about-nodeshift[nodeshift].
+
+. Find the name of your pod and follow the logs to watch it start.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+....
+$ oc logs -f pod/POD_NAME
+----
++
+IMPORTANT: After your pod has started, keep following the logs.
+
+. Test your application. 
++
+For example, if you had debug logging in the xref:mission-http-api-nodejs[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. Return to your pod logs and notice there are no debug logging messages in the logs. 
+
+. Set the `DEBUG` environment variable in your application pod to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env pod/POD_NAME DEBUG=mybooster
+----
+
+. Return to your pod logs to watch update roll out.
++
+IMPORTANT: After your update has rolled out, keep following the logs. This is where your debug logging will appear.
+
+. Test your application to invoke debug logging. 
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. View your application logs to see your debug messages.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+...
+mybooster name: Sarah +3m
+----
+
+
+
+To disable debug logging, remove the `DEBUG` environment variable from the pod:
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env pod/POD_NAME DEBUG-
+----
+
+More details on environment variables are avialable in the link:https://docs.openshift.com/online/dev_guide/environment_variables.html[OpenShift documentation].
```

Comment:
This should be in a _Related information_ section (on the same level as _Prerequisites_ and _Procedure._